CHANGELOG.md

```diff
@@ -1,4 +1,48 @@
-# Changelog
+# CHANGELOG
+
+## 1.25.0 (2022-04-11)
+
+MLflow 1.25.0 includes several major features and improvements:
+
+Features:
+
+- [Artifacts] Introduce a new `mlflow.artifacts.download_artifacts()` API that mirrors functionality of CLI `download_artifacts` (#5585, @dbczumar)
+- [Models] Add support for saving artifact if destination is an empty directory. (#5572, @akshaya-a)
+- [Models] Introduce support for `diviner` as a flavor. (#5553, @BenWilson2)
```

Comment:
Can we add a link to Diviner and a short (1 key phrase) explanation of what it is?

Reply:
docs link added